docs/user-guide/dual-mesh.ipynb

```diff
@@ -0,0 +1,353 @@
+{
```

Comment:
<p>"When constructing the dual mesh paired with a face centered variable, the data becomes node centered. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/UXARRAY/uxarray/pull/859/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>